lib/models/ui.js

```diff
@@ -0,0 +1,34 @@
+'use strict';
+
+const UI = require('console-ui');
+
+let singletonUI;
+
+/**
+ * The class that serves as a singleton wrapper for `console-ui`.
+ *
+ * @private
+ * @module ember-cli
+ * @class CommandLineUI
+ * @constructor
+ */
+class CommandLineUI {
+  constructor(args) {
+    let options = args || {};
+
+    if (!singletonUI) {
+      singletonUI = new UI({
+        inputStream: options.inputStream,
```

Comment:
In this location, https://github.com/ember-cli/ember-cli/pull/6577/files#diff-3677132bf6240059b3a3637716019c84R613, it used to use `process` in the fallback, but now it uses null options.